exchange/src/utils/logger/logger.cpp

```diff
@@ -17,35 +18,50 @@ Logger::get_logger()
     return logger;
 }
 
+std::string
+timestamp_in_ms()
+{
+    using namespace std::chrono;
+
+    auto now = system_clock::now();
+
+    auto nowAsTimeT = system_clock::to_time_t(now);
+
+    auto nowAsTm = *std::localtime(&nowAsTimeT);
+
+    auto ms = duration_cast<milliseconds>(now.time_since_epoch()) % 1000;
+
+    // Use stringstream to format the time
+    std::ostringstream timestamp;
+    timestamp << std::put_time(&nowAsTm, "%Y-%m-%d %H:%M:%S");
+    timestamp << '.' << std::setfill('0') << std::setw(3) << ms.count();
+
+    return timestamp.str();
```

Comment:
```suggestion
    using namespace std::chrono;

    // Get time from Quill
    auto now = quill::Clock::now();

    // Get epoch time in milliseconds, and create time point
    auto epoch_millis = duration_cast<milliseconds>(now.time_since_epoch());
    auto system_now = time_point<system_clock>{epoch_millis};

    return fmt::format("{:%Y-%m-%d %H:%M:%S}", system_now).substr(0, 23);
```

https://godbolt.org/z/oK46fK1xa